index.html

```diff
@@ -0,0 +1,25 @@
+<html>  
+  <head>
+    <title>Web Calculator</title>
+    <script src="script.js"></script>
+  </head>
+    <body>
+      <h1 style="text-align:center">Web Calculator</h1>
```

Comment:
Your mixing inline styling with css. You can technically do this, but don't. You should only use a seperate CSS file in vanilla HTML (everything you are doing now is vanilla html).